### ISSUE TITLE
sol.hpp missing #include <limits>

### ISSUE THREAD

**Issue by jmedding** [X-Friese/FlyWithLua#92]
Hi,

I tried compiling the project on Ubuntu and it failed. In the end, I had to add

`#include <limits>`

to sol.hpp above the lines that were giving the error.

Please see this StackOverflow for the details.
[https://stackoverflow.com/questions/71296302/numeric-limits-is-not-a-member-of-std](https://stackoverflow.com/questions/71296302/numeric-limits-is-not-a-member-of-std)

**Comment by sparker256** [comment]
I wonder what is different on your system. I build with Ubuntu 20.04.3 LTS and GitHub actions uses the same version and has no issue so a little confused. I have also tested with a newer version of sol and wonder if this will work for you
[sol.zip](https://github.com/X-Friese/FlyWithLua/files/8158780/sol.zip)
.

**Comment by jmedding** [comment]
Thanks for checking.  I never do any C or C++ development, so my setup is probably wrong. I looked again and infact my version of sol.hpp does have an `#include <limits>`, but it's below the code section that was giving the error.  The new file that you provided seems to be the same, so I have no idea why it works for you and not me. If it works for you then no need to fix it, but perhaps having this issue posted might help some others.

**Comment by sparker256** [comment]
If all the required software has been installed it would be good to see your full build log without your patch to see why this is happening. This is why we went to GitHub actions to make sure that the code will build on all platforms with each new push.

**Comment by jmedding** [comment]
I couldn't find a build log file, so I copied the output from the terminal into a text file.

[Hope it helps](https://www.dropbox.com/s/02el5k4rdo10xxc/failed_build_term_output.txt?dl=0)

If you tell me hot to generate a log file, I'll try.  I did some googling, but to no avail.

**Comment by sparker256** [comment]
That is what I was looking for the output from the terminal thanks.

**Comment by sparker256** [comment]
I think I have this fixed with the current version I just pushed. Could you check to see if that is true?

**Comment by jmedding** [comment]
I followed the install instructions for Ubuntu and it compiled as expected.

I placed the new FlyWithLua folder in my plugins folder and started XP11.  

Everything seems to work as expected, 

I consider this issue to be resolved.

I almost filed another issue since I couldn't figure out how to get that popup to go away.  The hint text running across the screen was very small and it tended to blend in with the background.  Just saying, you might get some issues from this if you don't make it a little more obvious what the user has to do.
